src/v/cluster/tm_stm.cc

```diff
@@ -181,6 +181,20 @@ ss::future<tm_stm::op_status> tm_stm::exclude_hosted_transaction(
     }
 }
 
+ss::future<tm_stm::op_status>
+tm_stm::set_draining_transactions(model::term_id term, draining_txs draining) {
+    if (!_hosted_txes.inited) {
+        co_return op_status::unknown;
+    }
+    auto new_hosted_tx = _hosted_txes;
+    auto error = new_hosted_tx.set_draining(draining);
+    if (error != tm_tx_hash_ranges_errc::success) {
+        co_return op_status::not_found;
```

Comment:
non success != not found

Reply:
It should unknown. not_found has a specific meaning while unknown may mean anything.